x/staking/keeper/delegation.go

```diff
@@ -49,7 +49,7 @@ func (k Keeper) GetAllDelegations(ctx sdk.Context) (delegations []types.Delegati
 }
 
 // return all delegations to a specific validator. Useful for querier.
-func (k Keeper) GetValidatorDelegations(ctx sdk.Context, valAddr sdk.ValAddress) (delegations []types.Delegation) {
+func (k Keeper) GetValidatorDelegations(ctx sdk.Context, valAddr sdk.Address) (delegations []types.Delegation) {
```

Comment:
undo

Reply:
Although semantically correct, we lose context here. I think in cases like this, we should use explicit types. You'd never pass a non-validator address here.